helm/templates/deployment.yaml

```diff
@@ -42,7 +42,7 @@ spec:
           env: {{ toYaml .Values.env.database | nindent 12 }}
       initContainers:
         - name: person-schema-creation
-          image: quay.io/ukhomeofficedigital/postgres:latest
+          image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/callisto/postgres:latest
```

Comment:
Do we want to move the url into the values files?

Reply:
Fixed